models/models.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+
+from pydantic import BaseModel
+
+
+class CreateMaskinportenClientBody(BaseModel):
+    name: str
+    description: str
+    scopes: List[str]
+
+
+class MaskinportenClient(BaseModel):
+    client_id: str
+    name: str
+    description: str
+    scopes: List[str]
+
+
+class ClientKey(BaseModel):
+    key_id: str
+    key: str
+
+
+class ClientKeyMetadata(BaseModel):
+    key_id: str
+    client_id: str
```

Comment:
Samme her, `In` og `Out`? Eller noe annet som gjør det tydeligere hva modellene brukes for?

Eller kan man i FastAPI ha én fellesmodell (`MaskinportenClient` og `ClientKey`) med alle feltene på seg, men heller ha ulike serializers for å begrense feltene som brukes til input og output i de forskjellige endepunktene?

Reply:
Siden deler swagger-dokumentasjonen request|response-models genereres ut ifra dataklassene (e.g. `response_model=ClientKey`) så burde det være separate klasser. De to klassene her er ikke et In/Out case slik som ovenfor, begge er i forstand Out hvor den ene returneres når man oppretter en ny nøkkel for en klient, mens den andre modellen brukes for å liste ut hvilke nøkler som finnes for en klient